cmd/osv-scanner/main_test.go

```diff
@@ -1520,6 +1520,48 @@ Filtered 2 vulnerabilities from output
 	}
 }
 
+func TestRun_SubCommands(t *testing.T) {
+	t.Parallel()
+	tests := []cliTestCase{
```

Comment:
shall we also test without any subcommand and default to `scan`?

Reply:
I didn't add tests for without subcommand as all other existing tests are doing that.